### DISCUSSION TITLE
Find result indicators - show number amount of items found

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#4007] [Ideas]
Suggesting to add the number amount of keyword items found in search 
`# of #` 
or 
`# / #`
to inform user how many instances have been found in current document.
These numbers also informs user of how many more instances there are left when navigating through them that they are coming to the end of the instances so they know the repeat find will occur.

Not sure if this should be split into two separate posts, I can if someone mentions it.
These numbers would be added to both....
-the search dialog box
-the toolbar search field

I think this one could be very useful, hopefully others agree.
This request goes well with the other feature request that I made recently which can be found here if anyone is interested #4008

Thank You

**Comment by elextr** [comment]
Find finds the next occurrence of the search setting and stops searching at that next occurrence.  It does not normally search the whole document so there is no count to be shown.  All the occurrences of the search setting is only found when the "Mark" search is requested which will search the whole document and mark all the occurrences, and since it now has a count it is shown on the status bar `Found nnn matches for ".....".`

Geany does not follow searching the whole document by default as part of the somewhat wavey principle of keeping actions cheap.

**Comment by advice2020** [comment]
I had never tried "Mark" in search, I quickly tried it now that you have mentioned it, have to do some more testing, but seems like it does generate something similar to what I was looking for.
I forgot I also can use find in file which generates results in the messages window, which could be helpful to others reading this as well.
Does not generate a count but does give a list of found items.

Thank You

**Comment by elextr** [comment]
> I forgot I also can use find in file which generates results in the messages window, which could be helpful to others reading this as well.
> Does not generate a count but does give a list of found items.

The count is added at the end of the list.
